adapters/openx/openxtest/supplemental/multi-imp.json

```diff
@@ -7,7 +7,7 @@
         "banner": {},
         "ext": {
           "bidder": {
-            "unit": "banner-unit-1",
```

Comment:
Was this test case previously invalid? Doesn't look like it adhered to the unit json schema pattern.

Reply:
I don't recall if these were valid at some point in the past, but they are not now.